src/c_utils/apibench.cpp

```diff
@@ -19,11 +19,12 @@ extern "C" {
 
 static int my_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 
-  if (RedisModule_Init(ctx, "ft", REDISEARCH_MODULE_VERSION, REDISMODULE_APIVER_1) ==
+  if (RedisModule_Init(ctx, REDISEARCH_MODULE_NAME, REDISEARCH_MODULE_VERSION, REDISMODULE_APIVER_1) ==
```

Comment:
Where is it set if its not defined?

Reply:
It has a default of "ft" in `version.h`. I'm going to make some changes in this, as I don't want have too granular control in the compiler arguments (like, I just want to declare what we're building with the compiler options, and figure out the details in `version.h`).